Dockerfile.remix-ide

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:18.10 as builder
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+ENV NVM_DIR /usr/local/nvm
+ENV NVM_VERSION 0.34.0
+ENV NODE_VERSION 10.16.0
+
+USER root
+RUN apt-get update && apt-get install -yqq curl git python build-essential wget
+RUN yes | adduser --disabled-password remix && mkdir /app
+
+# Install base dependencies
+RUN apt-get update && apt-get install -y -q --no-install-recommends \
+        apt-transport-https \
+        build-essential \
+        ca-certificates \
+        curl \
+        git \
+        libssl-dev \
+        wget
+
+#Install NVM, NODE, NPM
+WORKDIR $NVM_DIR
+RUN curl https://raw.githubusercontent.com/creationix/nvm/v$NVM_VERSION/install.sh | bash \
+    && . $NVM_DIR/nvm.sh \
+    && nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+
+ENV NODE_PATH $NVM_DIR/versions/node/v$NODE_VERSION/lib/node_modules
+ENV PATH      $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
+
+#Now do remix stuff
+USER remix
+WORKDIR /home/remix
+
+RUN git clone https://github.com/ethereum/remix.git
+RUN git clone https://github.com/ethereum/remix-ide.git
+
+WORKDIR /home/remix/remix
+RUN npm install # RUN cd remix && npm ci -> Update to this when package-lock.json files are added
+RUN npm run bootstrap
+
+WORKDIR /home/remix/remix-ide
```

Comment:
`npm install && npm run build && npm run serve should be enough`
also it is better to pull the branch `remix_live` instead of `master`